controllers/users.js

```diff
@@ -1,3 +1,5 @@
+const axios = require("axios");
```

Comment:
any specific reason for using axios, in case you need this there is a utility named fetch created that uses axios internally please look into that

Reply:
Initially my plan was to use the fetch but there are some benefits which axios provides, and package was already installed it means somewhere other developers also uses this axios method .

Axios Benefits:
Simpler syntax: Axios uses a simple and consistent syntax for making HTTP requests, while Fetch requires more verbose code and has some quirks in its API.

Automatic handling of common errors: Axios automatically handles common errors such as timeouts, network errors, and HTTP error status codes, making it easier to write robust code.

Interceptors: Axios allows you to define interceptors that can modify requests or responses before they are sent or received, which can be useful for things like authentication or error handling.

Cross-browser support: Axios works in all modern browsers, including Internet Explorer 11, while Fetch has limited support in older browsers.

Cancelation: Axios allows you to cancel requests, which can be useful if a user navigates away from a page before a request completes.
So i think its good to use axios externally.

Reply:
+1, why Axios here?
We have built a wrapper for it already so we should be using that instead of directly importing

Reply:
As discussed, I got permission to use Axios from you because this is a one-time script anyway we are going to remove this.